features/addressing/managing_zones/editing_zone.feature

```diff
@@ -0,0 +1,63 @@
+@managing_zones
+Feature: Editing a zone
+    In order to change my my tax and shipping configuration
+    As an Administrator
+    I want to be able to edit a zone
+
+    Background:
+        Given the store operates in "France" and "Germany"
+        And the store also has country "United States"
+        And this country has the "Alabama" province with "AL" code
+        And this country has the "Arizona" province with "AZ" code
+        And the store has a zone "North America" with code "NA"
+        And the store has a zone "South America" with code "SA"
+        And I am logged in as an administrator
+
+    @ui @javascript
+    Scenario: Removing a country from a zone
+        Given the store has a zone "European Union" with code "EU"
+        And it has the "France" country member
+        And it also has the "Germany" country member
+        And I want to modify the zone named "European Union"
+        When I remove the "Germany" country member
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this zone should have only the "France" country member
+
+    @ui @javascript
+    Scenario: Removing a province from a zone
+        Given the store has a zone "United States of America" with code "USA"
+        And it has the "Alabama" province member
+        And it also has the "Arizona" province member
```

Comment:
In background i assign province to country. But in this context province is the zone member.